staging/src/github.com/kubeedge/beehive/pkg/core/module.go

```diff
@@ -30,6 +32,7 @@ func init() {
 type ModuleInfo struct {
 	contextType string
 	remote      bool
+	active      bool
```

Comment:
What is the significance of the newly added **active** field? Does it mean that a module was started normally?

Reply:
Yes, it means a module is started and running normally. In this PR it's `true` if `module.Start()` doesn't exits.